CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Zilliqa Coding and Review Guidelines
+
+1. All submitted **Pull Requests** are assumed to be ready for review.  
```

Comment:
:bulb: "draft" pull requests are intended to replace other designators. They are not yet a full replacement, but it may be good to at least mention that draft pull requests are considered "in progress" even if the title doesn't say wip.

Reply:
yes, but a draft pull request if it does not have the specific tags, can be started to review, according to our guideline